Documentation/install/upgrade.rst

```diff
@@ -17,6 +17,91 @@ It is assuming that Cilium has been deployed using standard procedures as
 described in the :ref:`k8s_concepts_deployment`. If you have installed Cilium
 using the guide :ref:`ds_deploy`, then this is automatically the case.
 
+Running a pre-flight DaemonSet
+==============================
+
+.. _pre_flight:
+
+To avoid a major disruption in pods that have L7 policies enabled, we suggest
+to run a pre-flight Cilium `DaemonSet`. This pre-flight will make sure the Cilium
```

Comment:
It'd be helpful to elaborate on why this disruption occurs and why this solution remediates the problem. It's not immediately obvious why this would fix such problems.